search.go

```diff
@@ -311,6 +322,12 @@ func (r *SearchRequest) SortByCustom(order search.SortOrder) {
 	r.Sort = order
 }
 
+// SetSortAfter sets the request to skip over hits with a sort
+// value less than the provided sort after key
+func (r SearchRequest) SetSortAfter(after []string) {
```

Comment:
feels like this should be named SetSearchAfter?  not a strong feeling, but a naming-consistency tickle appeared in my mind

Reply:
oops that was typo, good catch